src/release.py

```diff
@@ -357,6 +359,9 @@ def run():
         summary = commit["commit"]["message"]
         summary = summary.splitlines()[0]
         summary = summary[:80]
+        # Bug 1868455: While GitHub autolinking doesn't suport spaces, Bugzilla autolinking
+        # doesn't support hyphens.
+        summary = bug_name_prefix_regexp.sub(r"bug-\1", summary)
```

Comment:
Two things:

1. This needs to convert `bug-nnnnnnn` (hyphen) to `bug nnnnnnn` (space). We're trying to fix the output so that Bugzilla's autolinking works and it only works on `bug nnnnnnn` (space).
2. We only want to do this conversation when the subcommand is `make-bug`. If it's `make-tag`, we want to leave the output as is.

Reply:
> We only want to do this conversation when the subcommand is make-bug. If it's make-tag, we want to leave the output as is.
The tag message is displayed in both GitHub (on the Tags page) and in Bugzilla via [a call to `git show {tag}`](https://github.com/willkg/socorro-release-test/blob/6ad421d04066a31e64525df2a81783e2e4ff3f0d/release.py#L203), so I don't see how we avoid broken links in that case. Is the Bugzilla description more important to have links than the tag comment from `make-tag`?

Reply:
When we run `./bin/release.py make-bug`, that creates a link that you use to create a bug in bugzilla. It's what you see in the description like this:

https://bugzilla.mozilla.org/show_bug.cgi?id=1867844#c0

That text should have `bug nnnnnnn` (space) things which work with bugzilla autolinking.

When we run `./bin/release.py make-tag --with-bug=NNNNNNN`, that creates the annotated tag that gets pushed to GitHub. We don't need to fix the bug numbers for that text. It's true that it also generates text that we copy-and-paste into Bugzilla which we see in comment 1:

https://bugzilla.mozilla.org/show_bug.cgi?id=1867844#c1

However, the entire text is inside Markdown preformatted block, so Bugzilla doesn't do any autolinking.

Given that, we only need the `bug-nnnnnnn` (hyphen) to `bug nnnnnnn` (space) conversion when using `make-bug`.